### PR TITLE
lend fixes

### PR DESCRIPTION
**Pull request by timongll**

### CODE REVIEW

File: lend/src/components/StepsHeader/index.tsx [hunk 1]
```diff
@@ -59,18 +59,24 @@ const OpenAppButton = styled.div`
...
-        {/* uncomment on launch */}
-        <a href={"/app"}>
-          <OpenAppButton>
-            <ButtonText>OPEN APP</ButtonText>
-          </OpenAppButton>
-        </a>
-      </ButtonContainer>
+      {button && (
+        <ButtonContainer>
+          {/* uncomment on launch */}
```
Comment (leonkho) [inline lend/src/components/StepsHeader/index.tsx:72]: Could we remove this comment?

File: lend/src/hooks/usePoolsAPR.ts [hunk 2]
```diff
@@ -43,10 +53,12 @@ export const usePoolsAPR = () => {
           formatUnits(poolData.rewardPerSecond, 18)
         );
         const poolSize = parseFloat(formatUnits(poolData.poolSize, 6));
-        aprsTemp[pool] =
-          (supplyRate + (rewardPerSecond * RBNPrice * 31536000) / poolSize) *
-          100;
-        supplyAprsTemp[pool] = supplyRate / 100;
+        aprsTemp[pool] = isDefault[pool]
+          ? 7 + ((rewardPerSecond * RBNPrice * 31536000) / poolSize) * 100
```
Comment (leonkho) [inline lend/src/hooks/usePoolsAPR.ts:57]: We should have constants for these numbers: `7`, `31536000`, etc.

File: lend/src/hooks/usePoolsAPR.ts [hunk 3]
```diff
@@ -66,12 +78,12 @@ export const usePoolsAPR = () => {
     return {
       loading: isLoading,
       aprs: {
-        wintermute: 0,
-        folkvang: 0,
+        wintermute: 7,
```
Comment (leonkho) [inline lend/src/hooks/usePoolsAPR.ts:81]: Assuming this is the same constant used above?

File: lend/src/pages/LendPage.tsx [hunk 4]
```diff
@@ -282,7 +282,7 @@ const Header = ({ setWalletModal }: HeaderProps) => {
       <HeaderRow>
         <MarqueeCol md={12} lg={6}>
           {/* <StatsMarquee /> */}
```
Comment (leonkho) [inline lend/src/pages/LendPage.tsx:284]: Is the `StatsMarquee` still going to be used here? Otherwise we can remove it

Comment (timongll) [reply lend/src/pages/LendPage.tsx:284]: we will use these once we get higher values

**Review by leonkho — APPROVED** [review]

**Pull request merged**